README.md

```diff
@@ -2,4 +2,34 @@
 
```

Comment:
Did you check if all the results can be reproduced after code refactoring?

Reply:
Yes. I ran bert+roberta and bert+roberta+xlnet to evaluate the result. The results are very close to the previous result.

Reply:
did you check the single models?

Reply:
Done, checked all the single models, very close to the previous result.